src/view/popup/__tests__/calculateEmbodiedEmissions.test.ts

```diff
@@ -0,0 +1,24 @@
+import { CountryName } from "../../../data/constants/CountryEmissions";
+import { calculateEmbodiedEmissions } from "../utils/calculateEmbodiedEmissions";
+
+const mockDevice: Map<CountryName, number> = new Map([["iPhone 11", 1]]);
+
+const mockDevices: Map<CountryName, number> = new Map([
+    ["iPhone 11", 0.2],
+    ["iPhone 7", 0.2],
+    ["Google Pixel 5", 0.4],
+]);
+
+describe("calculateCarbon", () => {
+    it("country should return corresponding value", () => {
```

Comment:
can we replace country and countries to device and devices in this file please? :)